accepted/2021/utf8/requirements/README.md

```diff
@@ -0,0 +1,26 @@
+# Requirements
```

Comment:
Are these requirements for UTF-8 support in general, or something more specific?

Reply:
For UTF-8 support in general. We expect this will become the index page for the collection of requirements docs.